content/react/concepts/virtual-dom/virtual-dom.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Virtual DOM'
-Description: 'In React, Virtual DOM is a conceptual representation of the actual DOM object, like a lightweight copy. A virtual DOM object has the same properties as a real DOM object, but it lacks the real thing’s power to directly change what’s on the screen. Manipulating the DOM is slow. Manipulating the virtual DOM is much faster, because nothing gets drawn onscreen. Think of manipulating the virtual DOM as editing a blueprint, as opposed to moving rooms in an actual house. With Virtual DOM, when changes are made to the UI and theres a re-render, they are compared to the relevant branches in the DOM-tree. Then, the changes are applied to the DOM-tree. React Virtual DOM compared with the Real DOM The Virtual DOM is not to be confused with the Shadow DOM, or any other technology or concept. When you render a JSX element, every single virtual DOM object gets updated. This sounds incredibly inefficient, but the cost is insignificant because the virtual DOM can update so quickly. Once the virtual DOM has updated, then React compares the virtual DOM with a virtual DOM snapshot that was taken right before the update. By comparing the new virtual DOM with a pre-update version, React figures out exactly which virtual DOM objects have changed. This process is called “diffing.”'
+Description: 'In React, Virtual DOM is a conceptual representation of the actual DOM object, like a lightweight copy. A virtual DOM object has the same properties as a real DOM object, but it lacks the real thing’s power to directly change what’s on the screen. Manipulating the DOM is slow. Manipulating the virtual DOM is much faster, because nothing gets drawn onscreen. Think of manipulating the virtual DOM as editing a blueprint, as opposed to moving rooms in an actual house. With Virtual DOM, when changes are made to the UI and there is a re-render, they are compared to the relevant branches in the DOM-tree. Then, the changes are applied to the DOM-tree. React Virtual DOM compared with the Real DOM The Virtual DOM is not to be confused with the Shadow DOM, or any other technology or concept. When you render a JSX element, every single virtual DOM object gets updated. This sounds incredibly inefficient, but the cost is insignificant because the virtual DOM can update so quickly. Once the virtual DOM has updated, then React compares the virtual DOM with a virtual DOM snapshot that was taken right before the update. By comparing the new virtual DOM with a pre-update version, React figures out exactly which virtual DOM objects have changed. This process is called “diffing.”'
```

Comment:
Per our [entry template](https://github.com/Codecademy/docs/blob/195d731f45c186e95a7b1effb92d41e1d09afe58/documentation/entry-template.md?plain=1#L3), let's shorten this `Description` to (ideally) 150 characters or less.